### PR TITLE
Message

### PR DESCRIPTION
**Pull request by Kaz1022**
Issues with message_history.ejs not being able to post message.

### CODE REVIEW

**Review by aprilhuang825 — COMMENTED** [review]
Good work Kaz!

File: routes/messages.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+   JOIN users on sender_id = users.id
+   JOIN items on item_id = items.id
+   WHERE receiver_id = $1
+   ORDER BY sender_name, created_at ASC;
+    `;
+
+    db.query(queryString,[req.session.user_id])
+      .then(data => {
+        const messages = data.rows;
+        console.log(messages);
```
Comment (b1u3too) [inline routes/messages.js:21]: TODO: remove these old console.log statements once you're done using them for debugging. :)

File: views/mymessages.ejs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            <button type="submit">Mark as READ</button>
+          </form>
+          -->
+          <p>---------------------------------------</p>
+        </div>
+      </article>
+    <% } %>
+  </main>
+
+
```
Comment (b1u3too) [inline views/mymessages.ejs:45]: With the way you connected the routes, this section is no longer needed!

**Review by b1u3too — APPROVED** [review]
Fabulous work! Left a couple of comments with thoughts of what can be tweaked as we continue onward and upward!

**Pull request merged**